code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -129,6 +129,13 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	name = "Pluoxium"
 	fusion_power = 10
 
+/datum/gas/miasma
+	id = "miasma"
+	specific_heat = 0.01
```

Comment:
Change this to 20 to be in line with the standard air mix

Reply:
weird why can I resolve this

Reply:
I can too, we are all admins on coderbus now